content/docs/pipelines/sdk/component-development.md

```diff
@@ -314,12 +216,16 @@ three placeholders available:
 *   `{inputValue: Some input name}`   
     This placeholder is replaced by the **value** of the argument to the
     specified input. This is useful for small pieces of input data.
+*   `{inputPath: Some input name}`   
+    This placeholder is replaced by the auto-generated **local path** where the
+    system will put the input data passed to the component during the pipeline run.
+    This placeholder instructs the system to write the input argument data to a file and pass the path to that data file to the component program (insttead of passing the whole data).
```

Comment:
s/insttead/instead

Also, instead of "passing the whole data", how about: `passing all the data as a placholder value`

Reply:
I've reworded this phrase.

Reply:
But it might still be not great....